www/source/layouts/api_builder.slim

```diff
@@ -0,0 +1,4 @@
+<a class="return" style="background-color:#007FAB; display:block; text-align:center; width:228px; margin:1rem 0 1.5rem 16px; padding:8px 0; color:white; text-decoration:none ;font-size:14px; text-transform:uppercase" href='https://www.habitat.sh/docs/'>Return to Docs</a>
```

Comment:
can we make this just `/docs/` so that it remains in whatever environment you are viewing the api?

Reply:
done